volt_core/Cargo.toml

```diff
@@ -22,6 +22,7 @@ structopt = "0.3"
 tar = "0.4"
 thiserror = "1.0"
 tokio = { version = "1.5.0", features = ["full"] }
+junction = { path = "../junction" }
```

Comment:
Maybe this should be under `[target.'cfg(windows)'.dependencies]` instead? So it's windows only.

Reply:
Yep :)